sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -371,6 +379,25 @@ export class KeyClient {
     return this.createKey(name, keyType, options);
   }
 
+  /**
+   * The createOkpKey method creates a new OKP key in Azure Key Vault. If the named key
```

Comment:
```suggestion
   * The `createOkpKey` method creates a new OKP key in Azure Key Vault. If the named key
```

nit. Is this useful?

Reply:
Also, could we link to what is OKP? in case that’s useful as well 🙂

Reply:
We have them for every other key type, so it's for consistency.